src/Data/Version/Internal.purs

```diff
@@ -16,7 +17,8 @@ isDigit :: Char -> Boolean
 isDigit c = '0' <= c && c <= '9'
 
 isAsciiAlpha :: Char -> Boolean
-isAsciiAlpha ch = between 'a' 'z' (toLower ch)
+isAsciiAlpha =
+  between (codePointFromChar 'a') (codePointFromChar 'z') <<< toLowerSimple <<< codePointFromChar
```

Comment:
The `unicode` package now uses code points for everything; `toLower` now returns `Array CodePoint`, but given that we're working with ascii here I thought `toLowerSimple` would suffice (`CodePoint -> CodePoint`).

See:
https://github.com/purescript-contrib/purescript-unicode/pull/15